test/expected.html

```diff
@@ -22,3 +22,309 @@
 <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewbox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6"><path stroke-linecap="round" stroke-linejoin="round" d="M4.5 10.5L12 3m0 0l7.5 7.5M12 3v18"></path>
 </svg>
 
+<p><math><mtable><mtr><mtd><mrow><mrow><mrow><mrow><mi>a</mi>
```

Comment:
We really need the `<math>` tag to specify its XML namespace...

Reply:
There is an attribute `xmlns` which I added to the test of my fork. The SVG namespace is also handled this way.

There are a couple of rules that are not handled by the library. For instance some attributes are only allowed on specific tags, and it's the library user's responsibility to make sure this stuff is correct.

Reply:
`xmlns` is only needed when you want to write the MathML as a valid XML document. It is not needed if you are embedding it inside an HTML5 document: https://developer.mozilla.org/en-US/docs/Web/MathML/Authoring#mathml_in_html_pages

Reply:
@kentookura Ah, ok cool! So long as we can supply it in user-land, I'm happy.

Reply:
You can also define your own wrapper for it that automatically adds it:

```ocaml
let math_xml attrs children = math (xmlns :: attrs) children
```